src/pages/ThorChainLP/Position/Position.tsx

```diff
@@ -168,7 +168,7 @@ export const Position = () => {
 
   const { data: earnings, isLoading: isEarningsLoading } = useQuery({
     ...reactQueries.thorchainLp.earnings(position?.dateFirstAdded),
-    enabled: Boolean(position),
+    enabled: Boolean(position?.dateFirstAdded),
```

Comment:
Do we want to keep this for another PR with more details on this change? 🙏🏽

Reply:
yeah, it didn't really do what I wanted it to do anyways

Reply:
https://github.com/shapeshift/web/pull/6387/commits/aea848e7d6618281de13738f52807f8f631f2330